lib/flutter_chips_selector.dart

```diff
@@ -8,67 +8,80 @@ import 'package:flutter/scheduler.dart';
 import 'package:flutter/services.dart';
 
 typedef ChipsBuilder<T> = Widget Function(
-    BuildContext context, ChipsSelectorState<T?> state, T? data);
+  BuildContext context,
+  ChipsSelectorState<T?> state,
```

Comment:
Since you have changed the supertype of the state:
```suggestion
  ChipsSelectorState<T> state,
```

Reply:
changed